ext/standard/basic_functions.stub.php

```diff
@@ -768,6 +768,140 @@ function scandir(string $directory, int $sorting_order = 0, $context = UNKNOWN):
 function glob(string $pattern, int $flags = 0): array|false {}
 #endif
 
+/* exec.c */
+
+function exec(string $command, &$output = null, &$result_code = null): string|false {}
+
+function system(string $command, &$result_code = null): string|false {}
+
+function passthru(string $command, &$result_code = null): string|null|false {}
```

Comment:
I don't think this can return string?

Reply:
Yes, you are right, I forgot about the `type != 3` condition here